### ISSUE TITLE
Tasks

### ISSUE THREAD

**Issue by DangHa** [DangHa/Kaggle-HuBMAP-HPA#2]
- [ ] U-net with simple example
- [ ] U-net with shallow architecture running with HuBMAP training set
- [ ] Choosing hyper-parameters (architecture, learning rate, ...)
- [ ] Find some way to expand the data
- [ ] Write report
